.github/workflows/tests_and_cov.yml

```diff
@@ -57,7 +57,7 @@ jobs:
           key: ${{ env.pythonLocation }}-${{ hashFiles('setup.py') }}-${{ hashFiles('requirements-dev.txt') }}
 
       - name: Install the HTML 2 PDF renderer
-        run: sudo apt-get update || true && sudo apt-get install -y wkhtmltopdf
+        run: sudo apt-get update || true && sudo apt-get install -y wkhtmltopdf git
```

Comment:
why?

Reply:
git does not seem to be installed on the tester image for some reason?

Reply:
All of those explicit `git` installs can be removed as soon as we can install `flask-ldapconn` from pypi again.

Reply:
Understood! 👍🏻